themes/default/content/docs/intro/cloud-providers/_index.md

```diff
@@ -18,6 +18,7 @@ aliases: ["/docs/reference/clouds/"]
 * [Azure-Native]({{< relref "./azure" >}})
 * [AzureAD]({{< relref "./azuread" >}})
 * [Google Cloud]({{< relref "./gcp" >}})
+* [Native Google Cloud (GCP-Native)]({{< relref "./gcp-native" >}})
```

Comment:
Should it be just `GCP-Native` (similar to `Azure-Native`)? Or what's our prose here @infin8x

Reply:
Just `Google-Native` works, I think